test/loopback.test.js

```diff
@@ -631,4 +631,73 @@ describe('loopback', function() {
       });
     });
   });
+
+  describe('new remote method configuration', function() {
+    it('Remote method - isStatic specified in method name', function() {
```

Comment:
Oh, is this test checking both static and prototype methods? In general, it's better to have one test for each different test scenario ("one assert per unit test"). If you think writing (and maintaining) two tests is not worth it, then perhaps this would be a better test name?

```js
it('parses "isStatic" flag from method name', ...)
```